db/re/item_db.conf

```diff
@@ -86918,6 +87078,103 @@ item_db: (
 	Weight: 10
 },
 {
+	Id: 12873
+	AegisName: "TE_Potion_Box"
+	Name: "TE Potion Box"
+	Type: 2
+	Buy: 20
+	Weight: 100
+	Trade: {
+		nodrop: true
+		notrade: true
+		nocart: true
+		noselltonpc: true
+		nomail: true
+		noauction: true
+	}
+	Script: <" getitem TE_White_Slim_Potion,10; ">
+},
+{
+	Id: 12874
+	AegisName: "Frost_Giant_Blood"
+	Name: "Frost Giant Blood"
+	Type: 2
+	Buy: 20
+	Weight: 100
+	Script: <"
+		specialeffect2 EF_POTION_CON;
+		autobonus "{ heal -3000,0; bonus2 bAddRace,RC_Player,100; bonus2 bMagicAddRace,RC_Player,100; }",10,0,0,SI_GVG_GIANT;
```

Comment:
if this is rathena ->
https://github.com/rathena/rathena/blob/master/doc/script_commands.txt#L5377
`bonus_script "<script code>",<duration>{,<flag>{,<type>{,<status_icon>{,<char_id>}}}};`
```
bonus_script "{ heal -3000,0; bonus2 bAddRace,RC_Player,100; bonus2 bMagicAddRace,RC_Player,100; }, 10, 8, 1, SI_GVG_GIANT;
```
just guessing

autobonus is trigger when you attack something
its not meant for usable items